### PR TITLE
Featureflag client

### PR DESCRIPTION
**Pull request by gg4race**
add feature flags and tests on client

### CODE REVIEW

**Comment by gg4race** [comment]
test this please

File: Leanplum-SDK/Classes/Internal/Leanplum.m [hunk 1]
```diff
@@ -2670,4 +2672,12 @@ + (NSSet *)parseEnabledCountersFromResponse:(NSDictionary *)response
     return nil;
 }
 
++ (NSSet *)parseEnabledFeatureFlagsFromResponse:(NSDictionary *)response
```
Comment (e7mac) [inline Leanplum-SDK/Classes/Internal/Leanplum.m]: let's add a test for this method and a similar one for countAggregator

File: Example/Tests/Classes/LPFeatureFlagManagerTest.m [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+//
+//  Created by Grace on 9/18/18.
+//  Copyright © 2018 Leanplum. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "LPFeatureFlagManager.h"
+#import "Constants.h"
+
+/**
```
Comment (e7mac) [inline Example/Tests/Classes/LPFeatureFlagManagerTest.m]: let's get rid of these comments too, since they'll be confusing for someone new

**Comment by e7mac** [comment]
test this please

**Pull request merged**